Gemfile.lock

```diff
@@ -49,7 +49,7 @@ GEM
     thor (1.2.1)
 
 PLATFORMS
-  ruby
+  arm64-darwin-22
```

Comment:
Is this normal :thinking: ?

Reply:
Nope, I should have run `bundle update` instead of removing the `Gemfile.lock` files and running `bundle install` again to regenerate them.

The handling of platforms is a bit confusing in bundler. There are very lengthy discussions in this ticket about this: https://github.com/rubygems/rubygems/issues/4269#issuecomment-758564690

What was done in the monolith apparently was to run `bundle lock --add-platform arm64-darwin ruby x86_64-darwin x86_64-linux`  to declare explicit support for these platforms, which would give:

```diff
 PLATFORMS
+  arm64-darwin
   ruby
+  x86_64-darwin
+  x86_64-linux
```

We may want to follow that path, but it is a bit unclear to me what the benefits/drawbacks could be. In any case, if someone removes the `Gemfile.lock` file, we would need to run `bundle lock` again to set the platforms... since that info is not persisted in the `Gemfile` itself.

Note that in the Ruby Tanker Core SDK, we have:

```
PLATFORMS
  arm64-darwin-22
  x86_64-darwin-21
  x86_64-linux
```

Which is different again.

We might want to add the `ruby` platform here since we ship C code, that will probably not work on other Ruby interpreters than MRI (i.e. "C Ruby).

Maybe we should take some time to understand the implications of declaring platforms here and update our standards/guidelines to have a common/unified way of handling them.